test/Microsoft.Data.Analysis.PerformanceTests/Microsoft.Data.Analysis.PerformanceTests.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>disable</ImplicitUsings>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <GenerateProgramFile>false</GenerateProgramFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.5" />
```

Comment:
We should use the same version as the other performance projects - $(BenchmarkDotNetVersion)

Reply:
See:

https://github.com/dotnet/machinelearning/blob/main/test/Microsoft.ML.PerformanceTests/Microsoft.ML.PerformanceTests.csproj#L14C58-L14C83

Reply:
@JakeRadMSFT there is an issue with the DotNetBenchmark version 0.12.0, that is currently used in ML .Net. It crashes with  NotSupportedException("Unknown .NET Runtime") on .Net 6.0. 
However. upgrading solution to the latest 0.13.5 leads to build error as Obsolete methods from this library are used in Microsoft.ML.CpuMath.PerformanceTests. 

I am not able to build Native part of ML .Net on my laptop ( due to #6823 ), so I can't verify my changes in the CpuMath.PerformanceTests project (or check if it's working correctly on .Net 6.0). That's why I used the latest version only for the DataFrame project.

Could you please help me with building native dlls for the project?

Reply:
Yeah let me take a look!